### DISCUSSION TITLE
Welcome to DALLE-2-App Discussions!

### DISCUSSION THREAD

**Discussion by 1998code** [1998code/DALLE-2-App#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Yahya-khodr** [comment]
Hello, 
How can I import this library in my firebase angular app, it is not recognizing the module.
Can you help me with that ?
I tried to import it in the client and server side both are not working .. 
Thank you

**Reply by Yahya-khodr** [reply]
@1998code

**Reply by 1998code** [reply]
Hi @Yahya-khodr 
For the module issue, please open a new issue in
https://github.com/ezzcodeezzlife/dalle-node/issues
🙂

**Reply by Yahya-khodr** [reply]
No it's not an issue with the module, but when I'm importing the library in typescript it doesn't recognize it but when I'm importing it with JavaScript it works fine.
Should I still open an issue there ?
@1998code  sorry for that, Thank you

**Reply by 1998code** [reply]
You can open an issue there and see if someone can make it works with TS.
This repo is for the nextJS application only. @Yahya-khodr  🙂
